.github/workflows/php.yml

```diff
@@ -27,63 +27,64 @@ jobs:
       - name: PHP syntax checker 8.1
         uses: prestashop/github-action-php-lint/8.1@master
 
-  # Check the PHP code follow the coding standards
   php-cs-fixer:
     name: PHP-CS-Fixer
     runs-on: ubuntu-latest
     steps:
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
+      - uses: PrestaShop/.github/.github/actions/php-ci/php-cs@master
         with:
           php-version: '7.4'
 
-      - name: Checkout
-        uses: actions/checkout@v2.0.0
-
-      - name: Cache dependencies
-        uses: actions/cache@v4
-        with:
-          path: vendor
-          key: php-${{ hashFiles('composer.lock') }}
-
-      - name: Install dependencies
-        run: composer install
-
-      - name: Run PHP-CS-Fixer
-        run: ./vendor/bin/php-cs-fixer fix --dry-run --diff --using-cache=no --diff-format udiff
-
-  # Run PHPStan against the module and a PrestaShop release
   phpstan:
     name: PHPStan
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        presta-versions: ['1.7.6', '1.7.7', '1.7.8', 'latest']
+        presta-version: ['1.7.7.x', '1.7.8.x']
     steps:
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          path: ${{ github.event.repository.name }}
+      
+      - uses: PrestaShop/.github/.github/actions/php-ci/phpstan@master
         with:
           php-version: '7.4'
-
-      - name: Checkout
-        uses: actions/checkout@v2.0.0
-
-      # Add vendor folder in cache to make next builds faster
-      - name: Cache vendor folder
-        uses: actions/cache@v4
+          presta-version: ${{ matrix.presta-version }}
+          module-name: ${{ github.event.repository.name }}
+          phpstan-level: '5'
+          phpstan-config: 'tests/phpstan/phpstan-${{ matrix.presta-version }}.neon'
+          phpstan-version: '^0.12'
+          composer-version: '2.2.18'
+  
+  phpstan-1_7_6_8:
```

Comment:
Why do you run static analysis on a specific version of Prestashop ? PHPStan is a static analysis tool. Its goal should be to check PHP code inside this mobule. I don't know why there is so much complexity by injecting prestashop code base and adapt its version...